src/libs/auth.ts

```diff
@@ -126,10 +127,21 @@ export const signOut = (cause: SignOutCause = 'unspecified'): void => {
   azureCookieReadyStore.reset();
   getSessionStorage().clear();
   azurePreviewStore.set(false);
+
   const auth: AuthContextProps = getAuthInstance();
   revokeTokens()
     .finally(() => auth.removeUser())
     .finally(() => auth.clearStaleState());
+  const authState = authStore.get();
```

Comment:
I might suggest grabbing only what's necessary here to clarify that there's only field being preserved from the previous `authStore` state.

```suggestion
  const { cookiesAccepted } = authStore.get();
```